spec/controllers/downloads_controller_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe DownloadsController do
+  let(:user) { FactoryGirl.create(:admin) }
+  before do
+    sign_in user if user
+  end
+  let(:persister) { Valkyrie.config.adapter.persister }
```

Comment:
Nope! I want to be able to change the adapter in `valkyrie.yml` to make sure the download controller works still.